src/main/proto/event.proto

```diff
@@ -35,118 +36,229 @@ service EventStore {
 message ReadHighestSequenceNrRequest {
+    /* The Identifier of the Aggregate for which to load events */
     string aggregate_id = 1;
+
+    /* The Sequence Number of the first event expected */
     int64 from_sequence_nr = 3;
 }
 
+/* The highest Sequence Number found for the provided request*/
 message ReadHighestSequenceNrResponse {
-    int64 to_sequence_nr = 1;
-}
 
-message RetrieveClusterInfoRequest {
+    /* The sequence number of the latest event */
+    int64 to_sequence_nr = 1;
 }
 
+/* A confirmation to a request from the client */
 message Confirmation {
+
+    /* True when successful, otherwise false */
     bool success = 1;
 }
 
+/* Request describing the desire to read events for a specific Aggregate */
 message GetAggregateEventsRequest {
+
+    /* The identifier of the aggregate to read events for */
     string aggregate_id = 1;
+
+    /* The sequence number of the first event to receive */
     int64 initial_sequence = 2;
+
+    /* Whether a snapshot may be returned as first element in the stream */
     bool allow_snapshots = 3;
 }
 
+/* Request message to retrieve Snapshot Events for a specific Aggregate instance */
 message GetAggregateSnapshotsRequest {
+
+    /* The identifier to fetch the snapshots for */
     string aggregate_id = 1;
+
+    /* The minimal sequence number of the snapshots to retrieve */
     int64 initial_sequence = 2;
+
+    /* The maximum sequence number of the snapshots to retrieve */
     int64 max_sequence = 3;
+
+    /* The maximum number of results to stream */
     int32 max_results = 4;
 }
 
+/* Request message to open an Event Stream from the Event Store. */
 message GetEventsRequest {
+
+    /* The token to start streaming from */
     int64 tracking_token = 1;
+
+    /* The number of messages the server may send before it needs to wait for more permits */
     int64 number_of_permits = 2;
+
+    /* The unique identifier of this client instance. Used for monitoring. */
     string client_id = 3;
+
+    /* The component name of this client instance. Used for monitoring. */
     string component_name = 4;
+
+    /* The name of the processor requesting this stream. Used for monitoring. */
     string processor = 5;
+
+    /* An enumeration of payload types that need to be blacklisted. The Server will stop sending messages of these
+       types in order to reduce I/O. Note that the Server may occasionally send a blacklisted message to prevent
+       time-outs and stale tokens on clients.
+     */
     repeated PayloadDescription blacklist = 6;
 }
 
+/* Message containing the information of an Event */
 message Event {
-    /* A message id */
+
+    /* The unique identifier of this event */
     string message_identifier = 1;
+
+    /* The identifier of the Aggregate instance that published this event, if any */
     string aggregate_identifier = 2;
+
+    /* The sequence number of the Event in the of the Aggregate instance that published it, if any */
```

Comment:
Typo: `...of the Event in the Aggregate instance...` i.o. `...of the Event in the of the Aggregate instance...`